sesam.py

```diff
@@ -939,7 +939,8 @@ def authenticate(self):
 
     def upload(self):
         if self.args.is_connector:
-            expand_connector(self.args.connector_dir, self.args.system_placeholder, self.args.expanded_dir)
+            expand_connector(self.args.connector_dir, self.args.system_placeholder, self.args.expanded_dir, self.args.profile)
+            self.authenticate()
```

Comment:
we can technically remove authenticate command now. It is merged with the upload. The intention to leave it untouched is for the case the user wants to only update secrets/environment variables without uploading the config.